### ISSUE TITLE
Issue with file view after upload

### ISSUE THREAD

**Issue by arjunrajlab** [Kitware/UPennContrast#698]
HIGH PRIORITY: issue with file view after upload.

Uploaded:
<img width="1718" alt="image" src="https://github.com/Kitware/UPennContrast/assets/13998824/6ac32f8e-c15a-4caf-8de3-eed610e08379">

Then I click "Submit" and get this:
<img width="1728" alt="image" src="https://github.com/Kitware/UPennContrast/assets/13998824/6542432b-f3d9-481e-9820-ea33116f235e">
(At this point, the little "disk" icon in the top right is pulsing. If I hover on it, the tooltip says "Saving…")

If I then click View Dataset, it does nothing and I get a server connection error:
<img width="1722" alt="image" src="https://github.com/Kitware/UPennContrast/assets/13998824/59db5ffd-0d37-4cbf-b86d-c934bf8b5ab4">

**Comment by arjunrajlab** [comment]
Oh wait, I figured out what was happening. It seems that it was putting the file in the base directory and not in my public folder. Perhaps we can make it default to the public folder so that files go in there unless we've navigated somewhere else? Or block the ability to put a file at this directory level?

<img width="601" alt="image" src="https://github.com/Kitware/UPennContrast/assets/13998824/6f5a36d6-be10-488e-8050-761eaeeb7886">
